test/Example.spec.js

```diff
@@ -4,10 +4,10 @@
 import chai from 'chai';
 const should = chai.should();
 
-import {implementations} from '../lib/main.js';
+import {allImplementations} from '../lib/main.js';
 
 describe('implementations', () => {
   it('should exist', async () => {
```

Comment:
Should additionally add back the test that checks if  `implementations` exist.

Reply:
I think we're ok as is here. We do need to expand the tests.